lib/compiler/passes/generate-js.js

```diff
@@ -156,12 +156,12 @@ function generateJS(ast, options) {
     function buildRegexp(cls) {
       return "/^["
         + (cls.inverted ? "^" : "")
-        + cls.value.map(part => Array.isArray(part)
+        + cls.value.map(part => (Array.isArray(part)
```

Comment:
I'd rather disable the eslint rule instead. IMO this code was completely fine.

Reply:
I actually like that it's clearer now.

Reply:
Ironically, I put the parens in the WRONG PLACE the first time. :)